figures/metrics.py

```diff
@@ -773,7 +773,7 @@ def get_monthly_history_metric(func, site, date_for, months_back,
             )
             history.append(dict(period=datetime.datetime.strftime(current_date, '%d-%m-%Y'), value=value, ))
             current_date = current_date + datetime.timedelta(days=1)
-        return dict(history=history)
+
```

Comment:
why is this change needed? we need the `history` object in response.

Reply:
This will give you a better picture. 
The return statement is added at the end already with the current_month variable.

Reply:
![image](https://user-images.githubusercontent.com/86109301/155976075-35564a10-4bf6-4e76-80e8-4c037b08ba65.png)